### PR TITLE
Update docs for tile release 1.3.0

### PR DESCRIPTION
**Pull request by jtuttle**

### CODE REVIEW

File: docs-content/release-notes.html.md.erb [hunk 1]
```diff
@@ -5,6 +5,48 @@ owner: Partners
...
+
+**Release Date:** August 9, 2021
+
+- CyberArk Conjur Service Broker - [v1.2.1](https://github.com/cyberark/conjur-service-broker/releases/v1.2.1)
+- CyberArk Conjur Buildpack - [v2.2.1](https://github.com/cyberark/cloudfoundry-conjur-buildpack/releases/v2.2.1)
+
+Features in this release:
+
+* Removed support for Conjur Enterprise v4
+  - We recommend migrating to Dynamic Access Provider v11+ or Conjur OSS v1+.
```
Comment (imheresamir) [inline docs-content/release-notes.html.md.erb]: Should references to DAP be changed to Conjur Enterprise?

Comment (jtuttle) [reply docs-content/release-notes.html.md.erb]: Hmm yeah I guess it should. I thought we had to leave this one in for some reason but I don't think that's the case. Will fix.

File: docs-content/release-notes.html.md.erb [hunk 2]
```diff
@@ -5,6 +5,48 @@ owner: Partners
...
+
+* Support for using Summon environments in secrets.yml file
+  - secrets.yml files can be split into sections by environment and secrets to
+    load at runtime can be specified using new SECRETS_YAML_ENVIRONMENT
+    variable. [cyberark/cloudfoundry-conjur-buildpack#44](https://github.com/cyberark/cloudfoundry-conjur-buildpack/issues/44)
+
+* Fixed error messages with invalid line numbers
+  - Caused by Cloudfoundry Buildpack encountering errors while parsing environment
+    variable settings after retrieving secrets from Conjur. [cyberark/cloudfoundry-conjur-buildpack#120](https://github.com/cyberark/cloudfoundry-conjur-buildpack/issues/120)
+
```
Comment (imheresamir) [inline docs-content/release-notes.html.md.erb:35]: Should there be a note about the addition of a `manifest.yml` "allowing for the usage of buildpack-packager and other native CloudFoundry features"? From the **Added** section of [2.1.6 changelog](https://github.com/cyberark/cloudfoundry-conjur-buildpack/compare/v2.1.5...v2.1.6)

**Review by imheresamir — APPROVED** [review]
LGTM!

**Pull request merged**